### PR TITLE
fix: black placeholder image

### LINKED ISSUE #3910: White K-logo placeholder on dark mode?
I thought there was PR fixing this that on
- light mode is light K 
- on dark mode is dark one? 

after switching through light and back to dark is ok

Happens on fresh deploy preview https://deploy-preview-3904--koda-nuxt.netlify.app/movr/explore/?page=1

image

### PR DESCRIPTION
**Pull request by KngZhi**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #3910
- [ ] Requires deployment <rubick/magick_issue>
- [ ] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer?target=DaWoVNNhBTdqJ5iPN3UcaLeamkPpJeGWdrs8LhvX9TQbadc)

### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

File: components/shared/view/BasicImage.vue [hunk 1]
```diff
@@ -28,7 +28,9 @@ defineProps({
 })
 
 const placeholder = computed(() => {
-  return $colorMode.preference === 'dark'
+  console.log($colorMode.preference)
```
Comment (prachi00) [inline components/shared/view/BasicImage.vue:31]: can we remove this console?

**Review by prachi00 — APPROVED** [review]

**Comment by petersopko** [comment]
@KngZhi not sure what happened here, but please, make sure our QA guild tests your PRs as well (don't remove them from reviewers)😆 
image

**Review by petersopko — APPROVED** [review]
wfm

**Comment by petersopko** [comment]
pay 20 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $20 @ 49.41 USD/KSM ~ 0.405 $KSM
🧗 [DaWoVNNhBTdqJ5iPN3UcaLeamkPpJeGWdrs8LhvX9TQbadc](https://kusama.subscan.io/account/DaWoVNNhBTdqJ5iPN3UcaLeamkPpJeGWdrs8LhvX9TQbadc)
🔗 [0x6186ba82350729e4bec5071ae88d6ddbc07435da75242c0e9c4dc1ddc792ab24](https://kusama.subscan.io/extrinsic/0x6186ba82350729e4bec5071ae88d6ddbc07435da75242c0e9c4dc1ddc792ab24)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Comment by KngZhi** [comment]
> @KngZhi not sure what happened here, but please, make sure our QA guild tests your PRs as well (don't remove them from reviewers)😆 image

don't know why that happen, just hit re-request review button, and it removes Lu 😂

**Pull request merged**